org.mwc.debrief.legacy/src/Debrief/ReaderWriter/GeoPDF/AbstractGeoPDFBuilder.java

```diff
@@ -382,15 +382,17 @@ public void setViewportArea(final WorldArea viewportArea) {
 	public static final String JAVASCRIPT_TEMPLATE_PATH = "/geopdf_animation.js";
 	public static final String JAVASCRIPT_TIMESTAMP_TAG = "!!JS_TIMESTAMPS";
 	public static final String JAVASCRIPT_TIMESTAMP_TAG_NON_INTERATIVE = "!!NONINTERACTLAYERS";
+	public static final String TIFF_SUFFIX = ".tiff";
 
 	public static final String NON_INTERACTIVE_SUFFIX = " (non-interactive)";
 
 	public static final String INTERACTIVE_LAYER_NAME = "Interactive Layers";
 
 	protected static File createBackgroundFile(final GeoPDFConfiguration configuration, final String background,
 			final ArrayList<File> filesToDelete) throws IOException, InterruptedException {
-		final File tmpFile = createTempFile("debriefgdalbackground" + UUID.randomUUID().toString().replaceAll("-", ""),
-				"", configuration.getTempFolder());
+		final File tmpFile = createTempFile(
+				"debriefgdalbackground" + UUID.randomUUID().toString().replaceAll("-", "") + TIFF_SUFFIX, "",
```

Comment:
@saulhidalgoaular - now that the output goes into a dedicated folder, we don't need to insert the random UUID, do we?  I guess we could use the layer name.  Again, this would help understanding/inspecting the data.